app/src/components/Common/TextInput/TextInput.tsx

```diff
@@ -10,33 +10,37 @@ interface IProps {
   onChange: (value: string) => void;
   isError?: boolean;
   onBlur?: () => void;
+  color?: 'White' | 'Gray' | 'Black';
 }
 
 export const TextInput = ({
   label,
   value,
   onChange,
   placeholder = '',
-  isNumber = false, 
+  isNumber = false,
   isError = false,
   onBlur = () => undefined,
+  color = 'Black',
 }: IProps): JSX.Element => {
   const [hasVisited, setVisited] = useState(false);
   const inputStyle = classNames(style.textInput, {
     [style.visited]: hasVisited,
     [style.error]: hasVisited && isError,
+    [style.blackTextInput]: color === 'Black',
+    [style.whiteTextInput]: color === 'White',
```

Comment:
Her kan det senere legges inn styling for grå felt som trengs i nye designet på registrering